db/sql/05_msar.sql

```diff
@@ -5740,6 +5740,16 @@ Returns:
 $$ LANGUAGE SQL RETURNS NULL ON NULL INPUT;
 
 
+CREATE OR REPLACE FUNCTION msar.raise_exception(err_msg text)
+RETURNS void AS $$/* 
+Utility function to raise an exceptions with an error message.
+*/
+BEGIN
+  RAISE EXCEPTION '%', err_msg;
+END;
+$$ LANGUAGE plpgsql RETURNS NULL ON NULL INPUT;
+
+
```

Comment:
Can you add a comment that this is for use in `SQL` functions where exception raising isn't otherwise possible?